lldb/include/lldb/Symbol/CompilerType.h

```diff
@@ -544,7 +545,7 @@ bool operator==(const CompilerType &lhs, const CompilerType &rhs);
 bool operator!=(const CompilerType &lhs, const CompilerType &rhs);
 
 struct CompilerType::IntegralTemplateArgument {
-  llvm::APSInt value;
+  clang::APValue value;
```

Comment:
I just realized this is adding a clang dependency on lldb core (non-plugin) code. What would you say to making this a `Scalar` instead?

Reply:
Ah good point. Changed to Scalar, wasn't a difficult transformation to make